nuxeo-ai-config/src/test/java/org/nuxeo/ai/TestAISearchREST.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *
+ * (C) Copyright 2020 Nuxeo SA (http://nuxeo.com/).
+ * This is unpublished proprietary source code of Nuxeo SA. All rights reserved.
+ * Notice of copyright on this source code does not indicate publication. *
+ *
+ *
+ * Contributors:
+ *     Nuxeo
+ */
+
+package org.nuxeo.ai;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
```

Comment:
shouldn't it be org.assertj.core.api.Assertions.assertThat, not Java6Assertions ?
you could use the common assertEquals here :)

Reply:
Oh youre right thank you! Im not a big fan of assertEquals and we did that across the projects already.